contracts/controller/src/views.rs

```diff
@@ -67,11 +67,31 @@ impl Contract {
     pub fn view_prices(&self, dtokens: Vec<AccountId>) -> HashMap<AccountId, Price> {
         self.get_prices_for_dtokens(dtokens)
     }
+
+    pub fn view_borrow_max(&self, user_id: AccountId, ticker_id: String) -> WBalance {
```

Comment:
Perhaps it's better to replace ticker_id with dtoken_id. It will simplify Price search and prevent changes LookupMap to UnorderedMap for prices in Contract state.